app/models/proposal.rb

```diff
@@ -203,9 +203,6 @@ def delegate_with_default(method)
     end
   end
 
-
-  ## delegated methods ##
-
```

Comment:
> we aren't using comments to organize methods

We are, actually...search for `###`.

> I would argue that we shouldn't have to if we have nice lean models!

:+1: Totally agree that having them in a standalone classes/models is better.